ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -162,6 +162,48 @@ def test_ip_access_to_server(self):
         self.boot_and_verify_node()
 
 
+class BaremetalIdracDriverDirectWholedisk(bsm.BaremetalStandaloneScenarioTest):
```

Comment:
"Wholedisk" can be removed from class name. Applies to all custom created classes.